documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,91 @@
+# Get Bootchart from a Clear Container #
+
+The following steps describe how to get a bootchart from a Clear Containers
+container. Currently `systemd` is the default init process in the Clear Containers mini-os
+system. The Systemd project provides a tool called `systemd-bootchart` that
+collects the CPU load, disk load, memory usage, as well as per-process information
+from the os boot process.
+
+## Configure Clear Containers to get a bootchart ##
+
+To get a bootchart from a Clear Container, follow the steps below.
+
+1. Prepare your Clear Container image
+
+The file `data/cc-bootchart.conf` can be used get a bootchart graph.
+
+The file must be installed in a Clear Container image with `systemd` and `systemd-bootchart`.
+
+The file `data/cc-bootchart` can be installed with the `configure` option
+`./autogen.sh --with-cc-image-bootchart-config-dir=DIR` where `DIR` is any of the following paths.
```

Comment:
This sentence must be removed. Please, instead list the explicit and complete commands for each path. The way it is written does not allow the reader to copy paste the commands into the console.